templates/cli_cmd.gotmpl

```diff
@@ -0,0 +1,315 @@
+// Code generated by go-openrpc. DO NOT EDIT.
+
+/*
+Copyright © 2019 NAME HERE <EMAIL ADDRESS>
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
```

Comment:
I don't think we should add licensing, but I would be interested to hear your thoughts?

Reply:
Agreed; I just had that in there as leftover from `cobra`.  Off the top of my head not sure if cobra's Apache 2 is "contagious" or not, but would be worth brushing up on.